app/Console/Commands/SendExpectedCheckinAlerts.php

```diff
@@ -44,25 +40,24 @@ public function __construct()
      */
     public function handle()
     {
-        $settings = Setting::getSettings();
+        $settings   = Setting::getSettings();
         $whenNotify = Carbon::now()->addDays(7);
-        $assets = Asset::with('assignedTo')->whereNotNull('assigned_to')->whereNotNull('expected_checkin')->where('expected_checkin', '<=', $whenNotify)->get();
+        $assets     = Asset::with('assignedTo')->whereNotNull('assigned_to')->whereNotNull('expected_checkin')->where('expected_checkin', '<=', $whenNotify)->get();
 
-        $this->info($whenNotify.' is deadline');
-        $this->info($assets->count().' assets');
+        $this->info($whenNotify . ' is deadline');
+        $this->info($assets->count() . ' assets');
 
         foreach ($assets as $asset) {
-            if ($asset->assigned  && $asset->checkedOutToUser()) {
+            if ($asset->assigned && $asset->checkedOutToUser()) {
                 $asset->assigned->notify((new ExpectedCheckinNotification($asset)));
             }
         }
 
-        // Send a rollup to the admin, if settings dictate
-        $recipient = new \App\Models\Recipients\AlertRecipient();
-
-        if (($assets) && ($assets->count() > 0) && ($settings->alerts_enabled && $settings->alert_email != '')) {
-            $recipient->notify(new ExpectedCheckinAdminNotification($assets));
+        if (($assets) && ($assets->count() > 0) && ('' != $settings->alert_email)) {
+            $recipients = collect(explode(',', $settings->alert_email))->map(function($item, $key) {
```

Comment:
We should maybe trim() here as well, in case people have extra spaces in their alert_emails field. `snipe@snipe.net,   snipe+foo@snipe.net` could potentially fail if we don't trim()

Reply:
The AlertRecipient class trims the string

Reply:
Ah, perfect, thank you :)